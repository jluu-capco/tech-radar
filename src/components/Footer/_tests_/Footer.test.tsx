import * as React from "react";
import { render } from "@testing-library/react";
import Footer from "../Footer";
import { BrowserRouter as Router } from "react-router-dom";

describe("Footer", () => {
  it("should render Footer component", () => {
    const { container } = render(
      <Router>
        <Footer />
      </Router>
    );
    expect(container.firstChild).toMatchSnapshot();
  });

  it("should check text content of Footer", () => {
    const { container } = render(
      <Router>
        <Footer />
      </Router>
    );

    expect(container.firstChild).toHaveTextContent(
      `Capco @${new Date().getFullYear()}All Rights Reserved`
    );
  });
});
