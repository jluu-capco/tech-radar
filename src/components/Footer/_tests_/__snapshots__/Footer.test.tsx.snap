// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer component 1`] = `
<div
  class="Footer__StyledFooter-sc-15feby7-0 gOlmXn footer"
>
  <img
    alt="Logo"
    height="44"
    src="logo.png"
  />
  <a
    href="https://www.capco.com/"
    rel="noreferrer"
    target="_blank"
  >
    Capco @
    2022
  </a>
  <div>
    All Rights Reserved
  </div>
</div>
`;
